apps/frontend/src/app/2_molecules/BannersCarousel/BannersCarousel.tsx

```diff
@@ -19,13 +21,19 @@ const responsive = {
   },
   tablet: {
     breakpoint: { max: 1199, min: 992 },
+    items: 3,
+    slidesToSlide: 1,
+  },
+  smallTablet: {
+    breakpoint: { max: 992, min: 768 },
     items: 2,
     slidesToSlide: 1,
   },
   mobile: {
-    breakpoint: { max: 992, min: 0 },
+    breakpoint: { max: 768, min: 0 },
```

Comment:
I think this is too huge on `768px`, I'd change this to `max: 576`, adjust min `smallTablet` value as well + styles so you see the arrows down to 576px.
<img width="765" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/eefaff60-35d5-4577-81f0-09cd77ff6dd0">

What do you think @soulBit?

Reply:
agreed!